File: jamcatch/jamcatch-assets/src/test/java/org/example/jamcatch/assets/test/JamCatchAssetsTest.java
package org.example.jamcatch.assets.test;

import static org.junit.jupiter.api.Assertions.assertFalse;
import static org.junit.jupiter.api.Assertions.assertNotNull;

import java.util.Set;
import org.example.jamcatch.assets.JamCatchAssets;
import org.example.javarca.model.Asset;
import org.example.javarca.model.AssetSet;
import org.junit.jupiter.api.Test;

public class JamCatchAssetsTest {

    @Test
    public void testAssetSetCreation() {
        AssetSet assetSet = new JamCatchAssets();

        assertNotNull(assetSet);
        Set<Asset> assets = assetSet.assets();
        assertNotNull(assets);
        assertFalse(assets.isEmpty(), "Asset set should not be empty");
    }
}

File: jamcatch/jamcatch-assets/src/test/kotlin/org/example/jamcatch/assets/test/JamCatchAssetsTest.kt
package org.example.jamcatch.assets.test

import org.example.jamcatch.assets.JamCatchAssets
import org.example.javarca.model.AssetSet
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class JamCatchAssetsTest {
    @Test
    fun testAssetSetCreation() {
        val assetSet: AssetSet = JamCatchAssets()

        Assertions.assertNotNull(assetSet)
        val assets = assetSet.assets()
        Assertions.assertNotNull(assets)
        Assertions.assertFalse(assets.isEmpty(), "Asset set should not be empty")
    }
}
